src/slices/GiveThunk.ts

```diff
@@ -204,6 +205,15 @@ export const changeGive = createAsyncThunk(
       if (giveTx) {
         segmentUA(uaData);
 
+        ReactGA.event({
```

Comment:
Looks like Segment.io analytics is set up? Probably want to use the Segment event tracking library OR GA (not both)

Reply:
we're going to be removing segment for sure & moving all in to ga

Reply:
I also don't think you can pass dimensions to events

Reply:
event args: https://github.com/react-ga/react-ga#reactgaeventargs

Reply:
Adding custom dimensions & metrics is supported in react-ga: 
https://github.com/react-ga/react-ga/blob/97d326ea9be48ec39027e989a680849aa140bc20/types/index.d.ts#L19